com.henriquesebastiao.confy.yml

```diff
@@ -0,0 +1,95 @@
+app-id: com.henriquesebastiao.confy
+default-branch: 1.0.5
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+command: confy
+build-options:
+  strip: false
+  no-debuginfo: true
+cleanup:
+  - /include
+  - /share/man
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --nosocket=wayland
+  - --share=network
+  - --device=dri
+  - --socket=pulseaudio
+  - --filesystem=xdg-documents
+modules:
+  - name: cpython
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/bin
+      - cp --archive support/python/bin/* /app/bin
+      - mkdir -p /app/include
+      - cp --archive support/python/include/* /app/include
+      - mkdir -p /app/lib
+      - cp --archive support/python/lib/* /app/lib
+      # Update installation prefix for pip package builds
+      - sed --follow-symlinks -i 's/prefix="\/install"/prefix="\/app"/g' /app/bin/python3-config
+      - sed --follow-symlinks -i 's/prefix=\/install/prefix=\/app/g' /app/lib/pkgconfig/python3.pc /app/lib/pkgconfig/python3-embed.pc
+    sources:
+      - type: dir
+        path: support/python
+        dest: support/python
+    cleanup:
+      - /bin/2to3*
+      - /bin/idle*
+      - /lib/python*/config-*
+  - name: bootstrap
+    no-autogen: true
+    sources:
+      - type: dir
+        path: src/bootstrap/
+  - name: resources
+    buildsystem: simple
+    build-commands:
+      - install -D com.henriquesebastiao.confy.desktop /app/share/applications/com.henriquesebastiao.confy.desktop
+      - cp --archive icons /app/share/icons
+    sources:
+      - type: file
+        path: com.henriquesebastiao.confy.desktop
+      - type: dir
+        path: icons/
+        dest: icons/
+  - name: upgrade_pip
+    buildsystem: simple
+    build-options:
+      build-args:
+        - --share=network  # For downloaded requirements.
```

Comment:
https://docs.flathub.org/docs/for-app-authors/requirements#no-network-access-during-build